lab_3_keywords_textrank/main.py

```diff
@@ -152,7 +171,11 @@ def decode(self, encoded_tokens: tuple[int, ...]) -> Optional[tuple[str, ...]]:
                 sequence of string tokens
         In case of out-of-dictionary input data, None is returned
         """
-        pass
+        for word in encoded_tokens:
+            if word not in self._id2word.keys():
+                return None
+        words = (self._id2word[token] for token in encoded_tokens)
+        return tuple(words)
```

Comment:
why do you need the `words` variable?

Reply:
I realized that I could just write it in the return string without implementing a new variable, so I've done it